src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -89,16 +84,10 @@ public static XmlDocument GetXmlDomFromHtml(string content, bool includeXmlDecla
 
 						// The regex here is mainly for the \s as a convenient way to remove whatever whitespace TIDY
 						// has inserted. It's a fringe benefit that we can use the[bi] to deal with both elements in one replace.
-						newContents = Regex.Replace(newContents, @"REMOVEWHITESPACE\s*\<([biu])\>", "<$1>");
-
-						//In BL2250, we still had REMOVEWHITESPACE sticking around sometimes. The way we reproduced it was
-						//with <u> </u>. That is, we started with
-						//"REMOVEWHITESPACE <u> </u>", then libtidy (properly) removed the <u></u>, leaving us with only
-						//"REMOVEWHITESPACE".
-						newContents = Regex.Replace(newContents, @"REMOVEWHITESPACE", "");
```

Comment:
Quite sure there's no case where this can still be needed? I suppose the critical difference is we now remove empty elements ourselves before tidy gets a chance?